lib/screens/overview_screen/widgets/my_health_section.dart

```diff
@@ -0,0 +1,199 @@
+/*
+*  This file is part of Picos, a health tracking mobile app
+*  Copyright (C) 2022 Healthcare IT Solutions GmbH
+*
+*  This program is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation, either version 3 of the License, or
+*  (at your option) any later version.
+*
+*  This program is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:picos/screens/my_medications_screen/my_medications_screen.dart';
+import 'package:picos/screens/physicians_screen.dart';
+import 'package:picos/screens/family_members_screen.dart';
+
+/// Widget which displays health-related information
+class MyHealthSection extends StatelessWidget {
+  // ignore: public_member_api_docs
```

Comment:
Using just a simple comment resolves the issue and is no more effort than the ignore statement.